ios/brave-ios/scripts/scheme_preaction.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Actions to run before every build in the brave-ios Xcode project
```

Comment:
offtopic, any reason you chose python over swift to write preactions?

Reply:
I actually didnt even think about that surprisingly, I choose python as thats what Chromium prefers when running scripts